backend/controllers/whereabouts/scheduledMoves.ts

```diff
@@ -24,6 +21,98 @@ export default ({ prisonApi }) => {
           text: values.description,
         }))
         .sort(sortTextAlphabetically),
+      courtEvents,
+    })
+
+  const index = async (req, res) => {
+    const { userDetails } = req.session
+    const { activeCaseLoadId } = userDetails
+    const date = req.query?.date || moment().format('DD/MM/YYYY')
+
+    const movementReasons = await prisonApi.getMovementReasons(res.locals)
```

Comment:
any reason for not using `Promise.all` here - would allow all three `await` events to be processed in parallel rather than having to wait for each in turn

Reply:
Agreed.